README.md

```diff
@@ -214,6 +215,17 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [raml2html](https://github.com/raml2html/raml2html) - Generates HTML documentation from a RAML file.
 * [ReDoc](https://github.com/Rebilly/ReDoc/) - OpenAPI/Swagger-powered three-panel documentation.
 
+## Management
+
+
+* [Kong](https://github.com/Kong/kong) - Scalable, distributed, and plugin oriented API gateway backed by Nginx.
+* [Tyk API Gateway](https://github.com/TykTechnologies/tyk) - Lightweight API gateway with analytics logging, written in Go.
+* [KrakenD](https://github.com/devopsfaith/krakend) - High performance API gateway with middlewares, written in Go.
+* [apiman](https://github.com/apiman/apiman) - Standalone and embeddable API management with web interface, written in Java.
```

Comment:
seems to be dead, please remove

Reply:
Although this one is not very popular, it's the only one that can be embedded. Besides, its community still contributing to it. Do you want to remove it?